### ISSUE TITLE
<footer> Element

### ISSUE THREAD

**Issue by joshsweaney** [xz/new.css#17]
To complement the <header> you can create, why not add some basic styling for a <footer> that must be at the bottom of the body, which makes a slightly darker footer that sticks to the bottom of the window? A sticky footer is one of those boilerplate bits of code that would be nice to have if a header is already there. It should have identical styling to the header.

**Comment by ZeroThe2nd** [comment]
Having it stick to the bottom of the screen or bottom of the full page?

Bottom of the page would make sense, whereas bottom of the screen would not. One can always opt to add a `display:sticky; bottom:0;` by themselves if someone would really want it to be sticky. Just adding the styling itself so one can add a footer should be sufficient.

**Comment by joshsweaney** [comment]
Sorry, I meant bottom of the page.

**Comment by 3x** [comment]
Good idea! Will be added later today.

**Comment by okeken** [comment]
@3x  has it been added?

**Comment by 3x** [comment]
> @3x  has it been added?

Ended up getting pushed back. This will be added in 1.3, which will hopefully released by Monday.

**Comment by pimhakkert** [comment]
Any update on this? @3x

**Comment by ZeroThe2nd** [comment]
@pimhakkert You can always make a PR yourself if you need the feature this much 😉

**Comment by pimhakkert** [comment]
@ZeroThe2nd That's true, but @3x already said it's scheduled for a later version. I don't want to get in the middle of that. I wouldn't consider myself good enough to do it anyways.

**Comment by ZeroThe2nd** [comment]
@pimhakkert Why not give it a try anyway? Give it a fork and try to implement it how you think is right, and make a PR. @3x can at the very least use it as a basis at the very least. Who knows, maybe you're better than you think. Don't undersell yourself 🙂
